src/main/java/mate/academy/model/User.java

```diff
@@ -33,4 +47,49 @@ public class User {
     private String lastName;
     @Column(name = "shipping_address")
     private String shippingAddress;
+    @ToStringExclude
```

Comment:
Use Lombok annotation if you need 
```suggestion
    @ToString.Exclude
```